Sources/ParseLiveQuery/Internal/ClientPrivate.swift

```diff
@@ -209,7 +209,7 @@ extension Client {
             let jsonEncoded = operation.JSONObjectRepresentation
             let jsonData = try NSJSONSerialization.dataWithJSONObject(jsonEncoded, options: NSJSONWritingOptions(rawValue: 0))
             let jsonString = String(data: jsonData, encoding: NSUTF8StringEncoding)
-
+            
```

Comment:
Looks like this is an unintentional whitespace change, probably added by Xcode?
Was added in commit 06222abfdf65c33d5fab6090b14c0a45966f611f and looks like never removed.

What I am referencing is all those whitespace only lines that didn't have whitespace and now they do.